stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/MessageEntity.kt

```diff
@@ -62,11 +63,22 @@ internal data class MessageInnerEntity(
     val shadowed: Boolean = false,
     /** if the message is also shown in the channel **/
     val showInChannel: Boolean = false,
-    /** if the message  **/
+    @Embedded(prefix = "channel_info")
+    val channelInfo: ChannelInfoEntity? = null,
```

Comment:
1. When this field is null Room saves all the inner nulls to the database
2. When the field is read, Room treat all the inner nulls as the whole object is null. That is, in that case `channelInfo` will be null